### ISSUE TITLE
docs - menu dropdown issue

### ISSUE THREAD

**Issue by Maxhodges** [Semantic-Org/Semantic-UI-React#776]
in the docs, clicking the on Categories toggles the dropdown once. Then it doesn't work again unless you refresh the page.

![image](https://cloud.githubusercontent.com/assets/957271/19849257/771bb43c-9f95-11e6-8859-a56010485ef5.png)

**Comment by layershifter** [comment]
Looks like #659

**Comment by Maxhodges** [comment]
when you click Categories in this demo, you can open and close the dropdown **once**. But then you cannot get it to reopen again, unless you refresh the page.

**Comment by levithomason** [comment]
@layershifter is correct, after the error is thrown the Dropdown stops working.  This is a duplicate of #659.

The console error:

```
app.6ff4071….js:6326 Uncaught TypeError: a._dropdown.blur is not a function(…)
```

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.64.4`.
